src/EditorFeatures/CSharpTest/Formatting/FormattingEngineTests.cs

```diff
@@ -2523,6 +2523,45 @@ interface I1<T> where T : I1<T>
             AssertFormatAfterTypeChar(code, expected, parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview));
         }
 
+        [WpfFact]
+        public void FormatUnsignedRightShift()
+        {
+            var code = @"$$
+class C
+{
+    public static C operator >>> ( C x, C y){
```

Comment:
nit: consider starting with no space or two spaces between `operator` and `>>>`, to show that part got formatted